.taskcluster.yml

```diff
@@ -1,245 +1,255 @@
+                                name: "Decision Task"
+                                description: 'The task that creates all of the other tasks in the task graph'
+                            else:
+                                $if: 'tasks_for == "action"'
+                                then:
+                                    name: "Action: ${action.title}"
+                                    description: '${action.description}'
+                                else:
+                                    name: "Decision Task for cron job ${cron.job_name}"
+                                    description: 'Created by a [cron task](https://tools.taskcluster.net/tasks/${cron.task_id})'
+                  provisionerId: "aws-provisioner-v1"
+                  workerType: "mobile-${level}-decision"
+                  tags:
+                      $if: 'tasks_for in ["github-push", "github-pull-request"]'
+                      then:
+                          kind: decision-task
+                      else:
+                          $if: 'tasks_for == "action"'
+                          then:
+                              kind: 'action-callback'
+                          else:
+                              $if: 'tasks_for == "cron"'
+                              then:
+                                  kind: cron-task
+                  routes: [checks]
+                  scopes:
+                      $if: 'tasks_for == "github-push"'
+                      then:
+                          - 'assume:repo:${repoUrl[8:]}:branch:${event.base.repo.branch}'
+                      else:
+                          $if: 'tasks_for == "github-pull-request"'
+                          then:
+                              - 'assume:repo:github.com/${event.pull_request.base.repo.full_name}:pull-request'
+                          else:
+                              $if: 'tasks_for == "action"'
+                              then:
+                                  # when all actions are hooks, we can calculate this directly rather than using a variable
+                                  - '${action.repo_scope}'
+                              else:
+                                  - 'assume:repo:${repoUrl[8:]}:cron:${cron.job_name}'
 
-    repository:
-      $if: 'tasks_for == "github-pull-request"'
-      then: ${event.pull_request.head.repo.html_url}
-      else: ${event.repository.html_url}
+                  requires: all-completed
+                  priority: lowest
+                  retries: 5
 
-    scheduler_id:
-      $if: 'tasks_for == "cron"'
-      then: focus-nightly-sched  # TODO: Rename to mobile-nightly-sched
-      else: taskcluster-github
+                  payload:
+                      env:
+                          # run-task uses these to check out the source; the inputs
+                          # to `mach taskgraph decision` are all on the command line.
+                          $merge:
```

Comment:
Nice `json-e`-fu on this `$merge`

Reply:
I think this predates me (copied from the m-c version of .taskcluster.yml).